Helper/OmiseHelper.php

```diff
@@ -292,4 +299,26 @@ public function createInvoiceAndMarkAsPaid($order, $chargeId, $isCapture = true)
         $invoice->setTransactionId($chargeId)->pay()->save();
         return $invoice;
     }
+
+    /**
+     * This method checks and return TRUE if $paymentMethod is
+     * offered by our Omise payment plugin otherwise returns false.
+     *
+     * @param string $paymentMethod
+     *
+     * @return boolean
+     */
+    public function isOmisePayment($paymentMethod)
+    {
+        $offsitePaymentMethods = array_merge($this->offsitePaymentMethods, $this->offsitePaymentImageCode);
+        $omisePaymentMethods = array_merge(
+            $offsitePaymentMethods,
+            [
+                Config::CODE,
+                Conveniencestore::CODE
```

Comment:
I think convenience store could also be a part of offsite payment methods. it must have been missed before 🤔